Sources/FluentUI_iOS/Components/Tab Bar/TabBarView.swift

```diff
@@ -92,11 +92,25 @@ open class TabBarView: UIView, TokenizedControl {
             stackView.setCustomSpacing(spacing, after: view)
         }
     }
+    /// Initializes MSTabBarView
+    /// - Parameter showsItemTitles: Determines whether or not to show the titles of the tab bar items.
+    @objc public convenience init(showsItemTitles: Bool = false) {
+        self.init(showsItemTitles: showsItemTitles, hideSeparator: false, disableBlur: false)
+    }
 
     /// Initializes MSTabBarView
     /// - Parameter showsItemTitles: Determines whether or not to show the titles of the tab bar items.
-    @objc public init(showsItemTitles: Bool = false) {
+    /// - Parameter hideSeparator: Removes the top divider displayed above the tab bar.
+    /// - Parameter disableBlur: Disables the blur effect applied to the tab bar.
+    @objc public init(showsItemTitles: Bool = false, hideSeparator: Bool = false, disableBlur: Bool = false) {
```

Comment:
would it be cleaner for these to be public properties instead of passing them in on init?